wrappers/python/tests/loss/test_xentropy.py

```diff
@@ -13,85 +13,117 @@
 
 import numpy as np
 import pytest
-import scipy.special as spsp
+import torch
 
 import nntile
 from nntile.loss import CrossEntropy as cross_entropy
 from nntile.tensor import TensorMoments
 
-config = nntile.starpu.Config(1, 0, 0)
+config = nntile.starpu.Config(1, 1, 0)
```

Comment:
if you ask for 1 CUDA device, then enable cublas by switching the last argument to `1`